test/mechanics/mesh/UnitMeshFem.cpp

```diff
@@ -28,6 +28,36 @@ void Check2DMesh(NuTo::MeshFem& mesh)
     BOOST_CHECK_NO_THROW(transformedMesh.NodeAtCoordinate(Eigen::Vector2d(43., 9.)));
 }
 
+BOOST_AUTO_TEST_CASE(MeshTrusses)
+{
+    auto mesh = NuTo::UnitMeshFem::CreateTrusses(13);
+    BOOST_CHECK_EQUAL(mesh.Elements.Size(), 13);
+    BOOST_CHECK_EQUAL(mesh.Nodes.Size(), 14);
+
+    bool nodesHaveExpectedValue = true;
```

Comment:
Why not check the condition directly without this `bool`?
~~~cpp
for(...) BOOST_CHECK(...);
~~~

Reply:
Because if you have messed something up, you get as many errors as you have nodes (for the same reason). Maybe there is a smart boost-way to give only one error message for checking members of an array, but I don't know it.

Reply:
With this bool variable, don't know how many nodes are wrong and you don't know which nodes are wrong. I think this would be useful information for a test...